### ISSUE TITLE
Protected terms/Add selected text  not bringing up previous lists

### ISSUE THREAD

**Issue by wujastyk** [JabRef/jabref#3161]
JabRef 4.0-dev--snapshot--2017-08-27--master--6a26798aa
Linux 4.8.0-58-generic amd64 
Java 1.8.0_144

1. In editing tab, title field,
2. select a word,
3. right click and choose "protected terms/Add selected text to list"

previous lists aren't shown, and can't be selected.  the only option is to create a new list.

Also, because one can't select an existing list, "Format Field" does nothing.

Problem disappears if one closes the editing window and re-opens it.

**Comment by Siedlerchr** [comment]
This only occurs when you inside the editor create a new list and then **immediately** want to assign terms to this new list. If you switch to an other entry it shows up. And next time you start JabRef, the list is shown. 
@lenhard  This sounds like one of these "mark as dirty" issues you recently worked in the entry editor. Any idea?

**Comment by lenhard** [comment]
To me this sounds more as if the list is passed to the entry editor when it is created. That would explain why it is only available when you switch to a new entry editor.

All in all this is not too critical, but probably also not too hard to fix.

**Comment by lenhard** [comment]
And I think I have fixed this.

@wujastyk Could you please try out the version available at http://builds.jabref.org/protected-terms-update/ and let me know if it works for you?

**Comment by wujastyk** [comment]
JabRef 4.0-dev--snapshot--2017-08-28--master--a88c20201
Linux 4.11.0-14-generic amd64
Java 1.8.0_144


I started today's JabRef on a different computer.  Went to editor field,
right clicked and chose "Protected terms".  I could select "New".  But I
couldn't find a way to tell JabRef that I already have a .terms list and I
want to use the one I've got.

I picked "new" and selected the existing file and was told it would be
overwritten.  But it wasn't.  When I came back to the field editor, I still
couldn't Protect terms.  So I closed the editor, and reopened it, and
*then* my .terms file was displayed.  I could add terms, and "format field."

Screenshot attached.



​
--
Professor Dominik Wujastyk <http://ualberta.academia.edu/DominikWujastyk>
​,​

Singhmar Chair in Classical Indian Society and Polity
​,​

Department of History and Classics <http://historyandclassics.ualberta.ca/>
​,​
University of Alberta, Canada
​.​

South Asia at the U of A:

​sas.ualberta.ca​
​​
